File: ez-finances-app/src/main/java/br/com/ez/app/web/v1/controller/HelloWorldController.java
package br.com.ez.app.web.v1.controller;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.ResponseStatus;
import org.springframework.web.bind.annotation.RestController;

/**
 * Created by raul.padua on 16/07/18
 */
@RestController
public class HelloWorldController {

    @GetMapping(path = "/v1/hello")
    @ResponseStatus(HttpStatus.OK)
    public String getCatalog() {
        return "Hello V1";
    }
}

File: ez-finances-app/src/main/kotlin/br/com/ez/finances/app/controller/HelloWorldController.kt
package br.com.ez.finances.app.controller

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.ResponseBody
import org.springframework.web.bind.annotation.ResponseStatus
import org.springframework.web.bind.annotation.RestController

@RestController
class HelloWorldController {

    @ResponseStatus(HttpStatus.OK)
    @ResponseBody
    @GetMapping("/hello")
    fun helloWorld() : String {
        return "HelloWorld"
    }
}